definitions/consented/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -4512,7 +4512,7 @@
     "CaseTypeID": "FinancialRemedyMVP2",
     "CaseFieldID": "provisionMadeForLabel",
     "UserRole": "caseworker-divorce-financialremedy-courtadmin",
```

Comment:
Label fields not needed CRUD Permission

Reply:
All label fields had ‘R’ permission, there is no harm adding CRUD permission to it.
As its label which can not be updated on CCD UI but when setting caseData we should have CRUD permission

Reply:
All label have R permission, there is no harm adding CRUD permission on it as its label